dynamic/informer/informer.go

```diff
@@ -105,10 +107,20 @@ type sharedResourceInformer struct {
 	close func()
 }
 
+type dynamicResourceClientListFunc func(metav1.ListOptions) (*unstructured.UnstructuredList, error)
+
+func castToCacheListFunc(fn dynamicResourceClientListFunc) cache.ListFunc {
+	options := metav1.ListOptions{}
```

Comment:
Is an empty `ListOptions` acceptable?

Reply:
In practice, I've only set the LabelSelector of a ListOption before. It might be good hygiene to set timeout but that will likely cause more problems on slow clusters that it would prevent.

So, I think an empty LabelSelector is alright.

Reply:
This is meant to just be an adapter function, so you should pass along the ListOptions from the caller. See my other comment below, which should make this clearer.